webapp/components/admin_console/email_authentication_settings.jsx

```diff
@@ -67,7 +67,7 @@ export default class EmailAuthenticationSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.email.allowEmailSignInTitle'
-                            defaultMessage='Allow Sign In With Email: '
+                            defaultMessage='Enable login with email: '
```

Comment:
Propose we use "sign-in" to match the UI, rather than "login" which isn't used in the UI?